src/Controls/tests/UITests/Tests/BorderUITests.cs

```diff
@@ -32,7 +32,7 @@ public void BordersWithVariousShapes()
 		{
 			App.WaitForElement("TargetView");
 			App.EnterText("TargetView", "BordersWithVariousShapes");
-			App.Click("GoButton");
+			App.Tap("GoButton");
```

Comment:
Until now, we only had the Click method that made tap on mobile and click on desktop on the indicated element. After adding Tap, the tests are updated because now we have a direct replacement for what we had in Xamarin.UITest https://learn.microsoft.com/en-us/dotnet/api/xamarin.uitest.android.androidapp.tap?view=xamarin-uitest-sdk.